static/devtron.css

```diff
@@ -275,6 +275,7 @@ pre,
 code {
   color: inherit;
   background-color: inherit;
+  -webkit-user-select: text;
```

Comment:
@kevinsawicki I just realized I couldn't copy/paste the list of APIs so I added this but want to double check you're 👍 on having `pre` and `code` be selectable. Seems like would be ok because those are things you may want to copy/paste anyways?